manage-participants/destroy.yml

```diff
@@ -15,19 +15,33 @@
       when:
         - ipa_host is defined
 
+- hosts: identity-hosts
+  name: Process Identity list
+  tasks:
+
+    - name: Add all users to removal queue
+      set_fact:
+        lodestar_identities_destroy: "{{ ( lodestar_identities_destroy | default([])) + [ { 'first_name': (identity.first_name | trim), 'last_name': (identity.last_name | trim), 'email': (identity.email | trim) , 'user_name': (identity.email.split('@')[0] | trim), 'state': 'absent' } ] }}"
```

Comment:
Please work on the readability a bit here - i.e.: newlines, etc. - alternatively look to see if you can perhaps define the user to remove as a yaml object first, e.g.:
```
set_fact:
  user_to_remove:
    first_name: "{{ identify.first_name | trim }}"
    last_name: ...
   : 
```

Reply:
Done